README.md

```diff
@@ -39,29 +38,21 @@ See the ["Metrics" section of CONTRIBUTING.md](./CONTRIBUTING.md#metrics).
 
 ## References
 
-For further reading, we encourage you to participate in the [InnerSource Commons site](https://paypal.github.io/InnerSourceCommons/)
-funded by PayPal where dozens of companies are already sharing experiences and working to
```

Comment:
When reading this, I was wondering if somewhere on https://innersourcecommons.org/ we should have a history section that acknowledges the companies that have enabled the birth and the initial growth of the ISC?

As eventually all of the living resources will have their references to PayPal removed, which seems a bit unfair.